guide/designing-products/user-research.md

```diff
@@ -115,7 +115,13 @@ Bitcoin only knows its users through pseudonymous addresses. We can analyze on-c
    layout = "float-left"
 %}
 
-While we often can’t know who is responsible for specific single transactions, we can still analyze overall network activity. No other currency in history has offered such a complete and accurate ledger of transactions. As designers, what can we learn from this data that allows us to improve the experience of our users.
+Bitcoin is a protocol with [layers](https://bitcoin.design/guide/getting-started/technology-primer/#do-all-transactions-have-to-be-this-secure) built on top of it.
+
+Layers built on top of the base layer, such as [Lightning network](https://bitcoin.design/guide/getting-started/technology-primer/#the-lightning-payment-network) make the specific transaction analysis impossible thanks to the [payment channels](https://bitcoin.design/guide/getting-started/technology-primer/#what-is-a-payment-channel).
```

Comment:
```suggestion
The base layer is a public record of transactions that can easily be analyzed. However, layers built on top, such as [Lightning network](/guide/getting-started/technology-primer/#the-lightning-payment-network) make analysis of specific transaction impossible, thanks to [payment channels](/guide/getting-started/technology-primer/#what-is-a-payment-channel).
```
I'd also describe the base layer, and not skip over it.

Reply:
Good catch, that makes a ton of sense, ty 👍